### PR TITLE
Update UI to use universal data API

### PR DESCRIPTION
**Pull request by chrisdostert**

### CODE REVIEW

File: webapp/src/constructDataUrl.ts [hunk 1]
```diff
@@ -1,7 +1,5 @@
-import path from 'path'
-
 export default function constructDataUrl(
     dataRef: string
 ): string {
-    return `http://localhost:42224/api/pkgs/${encodeURIComponent(path.dirname(dataRef))}/contents/${encodeURIComponent(path.basename(dataRef))}`
+    return `http://localhost:42224/api/data/${encodeURIComponent(dataRef)}`
```
Comment (apexskier) [inline webapp/src/constructDataUrl.ts:4]: Why doesn't this route show up in `sdks/go/node/api/urlTemplates.go`?

**Review by apexskier — COMMENTED** [review]
Can you add some background information to the PR description? I hadn't seen the "universal data API" this is referring to. Does anything else use it? Is the endpoint it's replacing (which looks deprecated) still used once this is merged?

File: webapp/src/constructDataUrl.ts (hunk 1 above)
Comment (chrisdostert) [reply webapp/src/constructDataUrl.ts:4]: it's in the API spec and is implemented but the path wasn't put into that file. It probably should be added.

Comment (chrisdostert) [reply webapp/src/constructDataUrl.ts:4]: It's meant to be the replacement to the `pkgs/{ref}` and `pkgs/{ref}/contents/{path}` endpoints.

**Review by apexskier — APPROVED** [review]

**Pull request merged**